src/components/Extensibility/components-form/MonacoRenderer.js

```diff
@@ -41,18 +41,17 @@ export function MonacoRenderer({
   required,
   resource,
   nestingLevel = 0,
-  originalResource,
 }) {
   const { tFromStoreKeys, t: tExt } = useGetTranslation();
 
   const value = getValue(storeKeys, resource);
   const jsonata = useJsonata({
-    resource: originalResource,
+    resource: resource,
```

Comment:
when the key and variable names are the same, you don't have to repeat them:
```suggestion
    resource,
```
similar to `value` two lines lower.